paasta_tools/cli/cmds/local_run.py

```diff
@@ -653,6 +690,98 @@ def check_if_port_free(port):
     return True
 
 
+def assume_aws_role(
+    instance_config: InstanceConfig,
+    service: str,
+    assume_role_arn: str,
+    assume_pod_identity: bool,
+    use_okta_role: bool,
+) -> AWSSessionCreds:
+    """Runs AWS cli to assume into the correct role, then extract and return the ENV variables from that session"""
+    pod_identity = instance_config.get_iam_role()
+    if assume_role_arn:
+        pod_identity = assume_role_arn
+    if assume_pod_identity and not pod_identity:
+        print(
+            f"Error: --assume-pod-identity passed but no pod identity was found for this instance ({instance_config.instance})",
+            file=sys.stderr,
+        )
+        sys.exit(1)
+    try:
+        with open("/nail/etc/runtimeenv") as runtimeenv_file:
+            aws_account = runtimeenv_file.read()
+            # Map runtimeenv in special cases to proper aws account name
+            if aws_account == "stage":
+                aws_account = "dev"
+            elif aws_account == "corp":
+                aws_account = "corpprod"
```

Comment:
something like:
```Python
RUNTIMEENV_TO_ACCOUNT_OVERRIDES = {
    'stage': 'dev',
    'corp': 'corpprod',
}
...
def assume_aws_role(
    ...
):
     ...
     with open("/nail/etc/runtimeenv") as runtimeenv_file:
            aws_account = runtimeenv_file.read()
            # generally, every runtimeenv maps to an account with the same name
            # ...but there are exceptions to every rule.
            RUNTIMEENV_TO_ACCOUNT_OVERRIDES.get(aws_account, aws_account)
```
might be a little neater, but this is fine as-is if you prefer the existing code